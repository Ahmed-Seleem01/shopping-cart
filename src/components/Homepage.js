// Import the 'Link' component from the 'react-router-dom' library and the 'homepage.scss' stylesheet
import { Link } from "react-router-dom";
import "../styles/homepage.scss";

// Define a functional component called 'Homepage'
const Homepage = () => {
  return (
    // Render a div element with the class name 'homepage'
    <div className="homepage">
      {/* Render a div element with the class name 'description' to hold information */}
      <div className="description">
        <h2>Ingrediento</h2>
        <p>Great taste starts here.</p>
        {/* Render a Link component that points to the '/shopping' route, when the button is clicked */}
        <Link to="/shopping">
          <button>Shop Now</button>
        </Link>
      </div>
    </div>
  );
};

// Export the 'Homepage' component as a default export
export default Homepage;
