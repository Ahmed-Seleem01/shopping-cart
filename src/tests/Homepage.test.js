import React from "react";
import { render, screen } from "@testing-library/react";
import Homepage from "../components/Homepage";

describe("check the snapshot", ()=>{
  it("check the component", ()=>{
    const {container} = render(<Homepage />)
    expect(container).toMatchSnapshot();
  })
})