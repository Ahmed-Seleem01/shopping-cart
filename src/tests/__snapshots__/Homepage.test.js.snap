// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`check the snapshot check the component 1`] = `
<div>
  <div
    class="homepage"
  >
    <div
      class="description"
    >
      <h2>
        Ingrediento
      </h2>
      <p>
        Great taste starts here.
      </p>
      <button>
        Shop Now
      </button>
    </div>
  </div>
</div>
`;
